packages/gatsby/cache-dir/head/head-export-handler-for-ssr.js

```diff
@@ -68,8 +70,17 @@ export function headHandlerForSSR({
           },
           node.childNodes[0]?.textContent
         )
-
-        validHeadNodes.push(element)
+        if (id) {
+          if (!seenIds.has(id)) {
+            validHeadNodes.push(element)
+            seenIds.set(id, validHeadNodes.length - 1)
+          } else {
+            const indexOfPreviouslyInsertedNode = seenIds.get(id)
```

Comment:
Why does the browser version include `validHeadNodes[indexOfPreviouslyInsertedNode].remove()` but SSR doesn't?

Reply:
`.remove` in browser is "just in case", we likely can drop it. SSR version is not rendering elements, it's rendering html strings, so it doesn't have potential DOM element to clean up - at most some no longere referenced strings/objects that can be GCed

Reply:
Ah that makes sense, saw all the other lines seemed duplicated 1 for 1 so wanted to double check. Thanks!